frontend/src/metabase/css/query_builder.css

```diff
@@ -489,17 +489,20 @@
   border-right: none;
 }
 
-.NativeQueryEditorDragHandle {
-  background: color(var(--color-border) blackness(+ 3%));
+.NativeQueryEditorDragHandleWrapper {
   position: absolute;
-  bottom: 0;
-  left: 50%;
   height: 5px;
```

Comment:
We might want to consider increasing the height every so slightly (maybe 7 or 8px instead of 5?). I was noticing it was a smidge finicky to grab on the first try.

Reply:
Just made this change. It's now 8px tall, but the visible handle is still 5px.